popolo/importers/popit.py

```diff
@@ -372,8 +369,7 @@ def update_membership(
             post_id_to_django_object,
             person_id_to_django_object,
     ):
-        Membership = self.get_popolo_model_class('Membership')
-        if 'id' not in membership_data:
+        def generate_membership_id(membership_data):
```

Comment:
It's a minor thing, but I think it'd be better to define this as a top level function in the module, rather than redefining it on each invocation of the method.